modules/machineset-creating-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,291 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * storage/persistent_storage/persistent-storage-azure.adoc
+// * storage/persistent_storage/persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "creating-machineset-azure"]
+:mapi:
+endif::[]
+ifeval::["{context}" == "persistent-storage-azure"]
+:pvc:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:pvc:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="machineset-creating-azure-ultra-disk_{context}"]
+= Creating machines on ultra disks by using machine sets
+
+You can deploy machines on ultra disks on Azure by editing your machine set YAML file.
+
+.Prerequisites
+
+* Have an existing Microsoft Azure cluster.
+
+.Procedure
+
+ifdef::mapi[]
+. Create a new custom secret using the worker-data-secret by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api get secret worker-user-data \
+--template='{{index .data.userData | base64decode}}' | jq > userData.txt
+----
++
+where #need to understand what these values are and where they come from#
```

Comment:
This is to leverage CoreOS Ignition, which allows manipulation of disks during initramfs(initiation phase) , the administrator needs to create a custom user-data configuration for this , the steps helps in doing that .